### PR TITLE
Proposed automation of charts from To-Do list

### PR DESCRIPTION
**Pull request by tnpb-encora**
please check if the application if the fields on the generated application is correct.

### CODE REVIEW

**Comment by brunomuniz-encora** [comment]
You should mark the activity as done in the README.md or remove the TODO line.

**Comment by brunomuniz-encora** [comment]
Don't you also need to remove the unused value from the example in the README.md file?

**Review by brunomuniz-encora — APPROVED** [review]

**Review by brunomuniz-encora — APPROVED** [review]

File: app-gen.py [hunk 1]
```diff
@@ -369,8 +369,9 @@ def _gen_fluxcd_manifest(self):
 
 
             # generate a manifest/static-overrides file
-            system_override_file = flux_dir + '/' + a_chart['name'] + '/' + a_chart['name'] + '-static-overrides.yaml'
-            open(system_override_file, 'w').close()
+            static_override_file = flux_dir + '/' + a_chart['name'] + '/' + a_chart['name'] + '-static-overrides.yaml'
+            with open(static_override_file, 'w') as f:
+                f.write('image:\n  repository: docker.io/brunomuniz/poc-starlingx')
```
Comment (Danmcaires) [inline app-gen.py]: why is this here?

Comment (brunomuniz-encora) [reply app-gen.py]: I suggested it as a placeholder until we figure this out for good. It lacks the `# Placeholder.` comment that I suggested, but that's the idea: to be a placeholder so the generated package "kinda" works.

Comment (Danmcaires) [reply app-gen.py]: But why put something related to a specific application in a generator that is to be used with any StarlingX application?

Comment (brunomuniz-encora) [reply app-gen.py]: Ok, I'm convinced, there's not much value in the placeholder, let's work to make the generator able to take care of this automatically anyway.

**Pull request merged**